benchmarks/routines/moe.py

```
@@ -2355,3 +2380,358 @@ def run_fp8_per_tensor_moe(
+                num_experts,
+                num_groups,
+                g_lora_a,
+                g_lora_b,
+                g_m_indptr,
+                g_shrink_out,
+                g_expand_out,
+            ),
+        )
+        gg_full_median = np.median(gg_full_times)
+
+    except (ImportError, Exception) as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)